### PR TITLE
Added in aat & preview config for em-ped-api

### PR DESCRIPTION
**Pull request by yogesh-hullatti**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EM-4597
### Change description ###
Added in aat & preview config for em-ped-api
**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by mdayican — APPROVED** [review]

File: apps/em/preview/base/kustomization.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@ kind: Kustomization
 resources:
   - ../../../base   # This is apps/base instead of apps/em/base as preview doesn't install everything
   - ../../identity/identity.yaml
+  - ../../em-ped-api/em-ped-api.yaml
```
Comment (adusumillipraveen) [inline apps/em/preview/base/kustomization.yaml:6]: this is preview.

Comment (yogesh-hullatti) [reply apps/em/preview/base/kustomization.yaml:6]: Yes. we need this for the PR's.

Comment (adusumillipraveen) [reply apps/em/preview/base/kustomization.yaml:6]: can u not use AAT ?

File: apps/em/em-ped-api/aat.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
```
Comment (adusumillipraveen) [inline apps/em/em-ped-api/aat.yaml:3]: this does nothing, unless u patch it somewhere.

Comment (yogesh-hullatti) [reply apps/em/em-ped-api/aat.yaml:3]: My understanding was this was the missing bit. In which case not sure why the PAI is not been deployed on AAT. Any clue?

**Pull request closed without merging**